docs/how-to/rocm-for-hpc/index.rst

```diff
@@ -0,0 +1,239 @@
+.. meta::
+   :description: How to use ROCm for HPC
+   :keywords: ROCm, AI, high performance computing, HPC, usage, tutorial
+
+******************
+Using ROCm for HPC
+******************
+
+The ROCm open-source software stack is optimized to extract the best high
```

Comment:
Please be wary about using terms like "best" 
You usually need data to back this up and then it can still be dicey.